tortoise/verifying_tortoise.go

```diff
@@ -237,19 +237,13 @@ func blockIDsToString(input []types.BlockID) string {
 // returns the local opinion on the validity of a block in a layer (support, against, or abstain)
 // TODO: cache but somehow check for changes (e.g., late-finishing Hare), maybe check hash?
 //    see https://github.com/spacemeshos/go-spacemesh/issues/2672
-func (t *turtle) getLocalBlockOpinion(ctx context.Context, layerID types.LayerID, blockid types.BlockID, inputs map[types.LayerID][]types.BlockID) (vec, error) {
+func (t *turtle) getLocalBlockOpinion(ctx context.Context, layerID types.LayerID, blockid types.BlockID, localOpinionsCache map[types.LayerID][]types.BlockID) (vec, error) {
```

Comment:
i think that this variable should refer to layer input vectors somehow, and no opinions.
for example this method returns support/abstain/against as an opinion

Reply:
changed